server/services/nuki/lib/commands/nuki.start.js

```
@@ -6,8 +7,12 @@ const logger = require('../../../../utils/logger');
  * @example
  * nuki.start();
  */
-function start() {
+async function start() {
   logger.debug(`Nuki: Starting`);
+  const { webOk, mqttOk } = await this.getStatus();
+  if (!webOk && !mqttOk) {
+    throw new ServiceNotConfiguredError('Not starting Nuki service, since neither Nuki Web API nor MQTT is configured');
+  }
   Object.values(this.protocols).forEach((handler) => handler.connect());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)